packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -394,31 +391,27 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
     }
   }
 
-  @autobind
-  protected onInputChange(value: string) {
```

Comment:
@Markionium @dzearing  Please avoid changing how public/protected members are defined in the future. This is technically breaking as anyone who overrides it (like I do) won't be able to compile when taking this version.